controllers/storagecluster/cephcluster_test.go

```diff
@@ -295,6 +296,46 @@ func TestNewCephClusterMonData(t *testing.T) {
 	}
 }
 
+func TestGenerateMgrSpec(t *testing.T) {
+	cases := []struct {
+		label        string
+		sc           *api.StorageCluster
+		isSingleNode string
+		expectedMgr  cephv1.MgrSpec
+	}{
+		{
+			label: "case 1", // Default case
```

Comment:
instead of case 1 and case 3 , the description can be more details about what you are testing.

Reply:
Done